client/packages/common/src/plugins/api/operations.generated.ts

```diff
@@ -0,0 +1,33 @@
+import * as Types from '@openmsupply-client/common';
+
+import { GraphQLClient, RequestOptions } from 'graphql-request';
+import gql from 'graphql-tag';
+type GraphQLClientRequestHeaders = RequestOptions['requestHeaders'];
+export type FrontendPluginMetadataQueryVariables = Types.Exact<{ [key: string]: never; }>;
+
+
+export type FrontendPluginMetadataQuery = { __typename: 'Queries', frontendPluginMetadata: Array<{ __typename: 'FrontendPluginMetadataNode', code: string, path: string }> };
+
+
+export const FrontendPluginMetadataDocument = gql`
+    query frontendPluginMetadata {
+  frontendPluginMetadata {
```

Comment:
Previous was:
```
query plugins {
  plugins {
    config
    name
    path
  }
```

Updated to "frontendPluginMetadata", sorry extra change I probably could have avoided